HTTPServer.test/MockConnection.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using HTTP_Server.core;
+
+namespace HTTPServer.test
+{
```

Comment:
We gotta talk about these braces. This is just the worst C#/Java convention ever. :)